app/src/main/java/com/connorlay/knitgrid/activities/PatternGridViewActivity.java

```diff
@@ -21,5 +26,17 @@ protected void onCreate(Bundle savedInstanceState) {
         transaction.replace(R.id.activity_pattern_grid_view_content_frame_layout, new PatternGridFragment());
         transaction.addToBackStack(null);
         transaction.commit();
+
+        if (isFirstLaunch()) {
```

Comment:
The only issue I have with that is that you need a context to get the SharedPreferences, so we'd have to pass that in, which kind of defeats the unit testing plus. What it sounds like we'd end up doing is making one of these objects in whatever ends up being our starting activity, calling this one method, and then throwing the object away. Seems like a lot of complexity and boilerplate for very little gain. Unless you can think of another way to accomplish this?